Framework/DataHandling/inc/MantidDataHandling/ISISRunLogs.h

```diff
@@ -56,12 +55,8 @@ class DLLExport ISISRunLogs {
   void addPeriodLog(const int i, API::Run &exptRun);
 
 private:
-  DISABLE_DEFAULT_CONSTRUCT(ISISRunLogs)
```

Comment:
I agree that this seems useless.

But shouldn't the default copy be disabled? We have a `unique_ptr` as member and should probably prevent shallow copies...?